examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java

```diff
@@ -46,8 +46,8 @@ public void onCreate(Bundle savedInstanceState) {
 
     // Set default address.
     // Android emulator forwards `10.0.2.2` to the host machine.
```

Comment:
Where does this `10.0.2.2` come from? Could you link to some documentation?

Reply:
Done